docs/guide/acquiring_data/jsoc.rst

```diff
@@ -512,6 +492,10 @@ you can download them by `~sunpy.net.fido_factory.UnifiedDownloaderFactory.fetch
 
     >>> downloaded_files = Fido.fetch(res)  # doctest: +SKIP
 
+To export a request for download, you must have used the `sunpy.net.jsoc.attrs.Notify` attribute at search time to specify your email address.
+Also **only whole queries can be downloaded from JSOC**, this means that no slicing operations performed on the results object will affect the results.
```

Comment:
I feel like this should be bigger, bolder and moved to the start?

Reply:
You mean under the "downloading data" heading?

Reply:
Sure or maybe after the first example?

Reply:
Well this is the first example which does a fetch, and you only need this if you are doing a fetch.

Reply:
Perhaps I should remake the query here as well to show this in action.